i18n-helpers/src/bin/markdown-gettext.rs

```diff
@@ -0,0 +1,248 @@
+use anyhow::{anyhow, bail, Context};
+use mdbook_i18n_helpers::translate;
+use polib::catalog::Catalog;
+use polib::po_file;
+use std::fs::File;
+use std::io::{BufReader, Read, Write};
+use std::path::PathBuf;
+use std::{env, fs};
+
+fn split_argument(arg: &String) -> Option<String> {
+    let splitted: Vec<_> = arg.split('=').collect();
+    if splitted.len() == 2 {
+        return Some(splitted[1].to_string());
+    }
+
+    None
+}
```

Comment:
I think it's time to use a real command line parsing library. Please take a look at how @djmitche did it in [mdbook-course](https://github.com/google/comprehensive-rust/blob/main/mdbook-course/src/main.rs).

Reply:
(fwiw, I adapted that from the example in the mdbook docs)